lib/ain-grpc/build.rs

```diff
@@ -266,6 +266,7 @@ fn compile_proto_and_generate_services(
         // There's no way to compile protos using custom generator in tonic,
         // so we're left with creating a prost config and using that for codegen.
         let mut config = Config::new();
+        config.protoc_arg("--experimental_allow_proto3_optional");
```

Comment:
Do we need this here ?

Reply:
yea.. i got asked to add the flag 😕 
```
Caused by:
  process didn't exit successfully: `/home/canonbrother/dev/DeFiCh/ain/build/lib/target/debug/build/ain-grpc-3d03a453500eafb3/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=/home/canonbrother/dev/DeFiCh/ain/lib/proto/types/tx.proto
  cargo:rerun-if-changed=/home/canonbrother/dev/DeFiCh/ain/lib/proto/types/eth.proto
  cargo:rerun-if-changed=/home/canonbrother/dev/DeFiCh/ain/lib/proto/types/block.proto

  --- stderr
  thread 'main' panicked at 'compiling protobuf: Custom { kind: Other, error: "protoc failed: types/eth.proto: This file contains proto3 optional fields, but --experimental_allow_proto3_optional was not set.\n" }', ain-grpc/build.rs:274:14
'``

Reply:
hmm.. wait.. seems this's blocking macos